Packages/Network/Sources/Network/DeepLClient.swift

```diff
@@ -12,20 +12,8 @@ public struct DeepLClient: Sendable {
     "https://api\(deeplUserAPIFree && (deeplUserAPIKey != nil) ? "-free" : "").deepl.com/v2/translate"
   }
 
-  private var APIKey: String {
```

Comment:
I think it's fine to remove the app since it now defaults to the Apple translation service on iOS 17.4. We will see how the users react.